src/overview/tooling-and-explorers/bridge.md

```diff
@@ -0,0 +1,119 @@
+---
+title: RLC Bridge
+description:
+  Bridge RLC tokens between networks to interact with the iExec protocol.
+  Transfer RLC to Bellecour (xRLC) and Arbitrum networks using dedicated bridges
+---
+
+# RLC Bridge
+
+**RLC** (RLC Token) is the essential cryptocurrency for interacting with the
+iExec protocol. Whether you are executing tasks, accessing protected data, or
+participating in the iExec decentralized confidential computing network, you
+will need RLC tokens on the appropriate network.
+
+This guide helps you bridge RLC tokens to **Bellecour** (becoming xRLC) and
+**Arbitrum** networks using dedicated bridge solutions.
+
+## 🎯 Why Bridge RLC?
+
+RLC tokens are required for various iExec protocol interactions:
+
+- **Task Execution**: Pay for confidential computing tasks
+- **Data Access**: Purchase access to protected datasets
+- **Application Usage**: Pay for iApp executions
+
+## 🗂️ Available Bridges
+
+iExec provides dedicated bridge solutions for seamless token transfers across
+networks:
+
+<CardGrid>
+  <ProjectCard
+    title="Bellecour Bridge"
+    description="Bridge RLC tokens between Ethereum mainnet and Bellecour sidechain (xRLC) in both directions"
```

Comment:
The end of the sentence is not visible

<img width="465" height="318" alt="image" src="https://github.com/user-attachments/assets/70be0d4d-2531-4100-a097-a07de26820c6" />